src/helpers/Typecast.php

```diff
@@ -118,6 +120,13 @@ private static function property(string $class, string $property, mixed &$value)
                     $value = $date ?: null;
                 }
                 return;
+            case is_subclass_of($typeName, self::TYPE_BACKED_ENUM):
```

Comment:
`switch` statements don’t allow for dynamic matching like this.

Reply:
In lieu of converting to a `match`, I just moved the logic to `default`